examples/linear_extension/linear_extension.jl

```diff
@@ -14,31 +14,31 @@ function run_linear_extension_example()
     # -------------------------------
     ## scalar parameters
     E = 2.0             # Young modulus in Pa
-    ν = 0.4             # Poisson's ratio
+    nu = 0.4             # Poisson's ratio
     p = 3               # Tension load in Pa
     tension(t) = p * t  # Tension load function
-    Lᵢ = 2.0            # Dimension in x of the box in m
-    Lⱼ = 1.0            # Dimension in y of the box in m
-    Lₖ = 1.0            # Dimension in z of the box in m
+    Lx = 2.0            # Dimension in x of the box in m 
+    Ly = 1.0            # Dimension in y of the box in m
+    Lz = 1.0            # Dimension in z of the box in m
     RTOL = 1e-4         # Relative tolerance for tests
     NSTEPS = 9          # Number of steps for the test
     ms = 0.5            # Refinement factor
     # -------------------------------
     # Materials
     # -------------------------------
     mat_label = "mat"
-    mat = IsotropicLinearElastic(E, ν, mat_label)
+    mat = IsotropicLinearElastic(E, nu, mat_label)
     s_materials = StructuralMaterial(mat)
     # -------------------------------
     # Boundary conditions
     # -------------------------------
     # Fixed dofs
     bc₁_label = "fixed-ux"
-    bc₁ = FixedDof(:u, [1], bc₁_label)
-    bc₂_label = "fixed-uj"
-    bc₂ = FixedDof(:u, [2], bc₂_label)
-    bc₃_label = "fixed-uk"
-    bc₃ = FixedDof(:u, [3], bc₃_label)
+    bc₁ = FixedDof([:u], [1], bc₁_label)
```

Comment:
This needs to be a scalar, since handling vector inputs was removed in a recent PR.

Reply:
done in [06d433a](https://github.com/ONSAS/ONSAS.jl/pull/344/commits/06d433a8d0a032a2c4867ed54d4817a927007e21) gracias!